lib/tasks/dhis2.rake

```diff
@@ -0,0 +1,61 @@
+namespace :dhis2 do
+  desc "Export aggregate indicators for each facility to DHIS2"
+  task export: :environment do
+    data_elements_map = {
```

Comment:
These are Bangladesh DHIS2 IDs (that I created). This whole thing is temporary; future iterations will have a separate data element config file

Reply:
Can you include a comment indicating that these are hard coded to SBX values for now?